movie-mlops-project/serving/model_loader.py

```
@@ -0,0 +1,340 @@
+            bucket_name: S3 버킷명
+            model_key: 모델 파일의 S3 키 (경로)
+            
+        Returns:
+            Optional[bytes]: 다운로드된 모델 파일의 바이트 데이터, 실패시 None
+        """
+        if not self.s3_client:
+            logger.error("S3 클라이언트가 초기화되지 않았습니다.")
+            return None
+        
+        try:
+            logger.info(f"📥 S3에서 모델 다운로드 시작...")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)